lib/aquilon/aqdb/model/building.py

```diff
@@ -34,8 +34,4 @@ class Building(Location):
 
     address = Column(String(255), nullable=False)
 
-    next_rackid = Column(Integer, nullable=False, default=2)
```

Comment:
Added by accident? Probably you need to rebase on latest upstream.

Reply:
This is a little bit strange, I thought I just reverted the previous commit which had something related to building.py. I started from the latest upstream if I'm right (latest=1 week ago!). May be the easiest is that I remove this line (or building.py) from the revert commit if this is the only change?